pkg/operator/ceph/csi/csi.go

```diff
@@ -315,11 +315,11 @@ func (r *ReconcileCSI) setParams() error {
 		disableVGS := strings.EqualFold(k8sutil.GetValue(r.opConfig.Parameters, "CSI_ENABLE_VOLUME_GROUP_SNAPSHOT", "true"), "false")
 		const (
 			enableVolumeGroupSnapshotFlag = "--enable-volume-group-snapshots="
-			featureGateFlag               = "--feature-gate=CSIVolumeGroupSnapshot="
+			featureGateFlag               = "--feature-gates=CSIVolumeGroupSnapshot="
```

Comment:
Thanks for fixing this typo :) , @travisn we need to backport this to downstream as well.